roles/dhcp-server/templates/local.conf.j2

```diff
@@ -8,3 +8,6 @@ dhcp-boot=pxelinux.0
 dhcp-authoritative
 enable-tftp
 tftp-root=/srv/tftp
+no-hosts
```

Comment:
that's a bit of a contradiction.   or redundancy.   it is either addressing a case that will never exist, or addressing the case where things are broken.  If /etc/hosts is in place, then we don't need dns. 

however, even with static, dns can serve everything by putting a 2nd *almost* copy of the data in a file.  almost = different IP for $server 

I like putting the name:IP data in it's own file that is just for the dns, not also /etc/hosts.  

Mainly because of waning $server:IP to be defined as something other than 127.0.1.1 like it is in /etc/hosts.